src/frontend-samples/clash-review-sample/ClashDetectionApis.ts

```diff
@@ -3,18 +3,16 @@
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 import { assert } from "@bentley/bentleyjs-core";
-import { IncludePrefix, request, Response } from "@bentley/itwin-client";
+import { AuthorizedClientRequestContext, IncludePrefix, request, RequestOptions, Response } from "@bentley/itwin-client";
 import { IModelApp } from "@bentley/imodeljs-frontend";
 import { IModelHubClient, VersionQuery } from "@bentley/imodelhub-client";
-import { NoSignInIAuthClient } from "NoSignInIAuthClient";
-import ClashReviewApp from "./ClashReviewApp";
+import { AuthorizationClient } from "@itwinjs-sandbox";
 
 export default class ClashDetectionApis {
 
   // Retrieves a list of Design Validation tests for the project specified by the project id.
   // https://developer.bentley.com/api-operations?group=administration&api=projects&operation=get-project-validation-tests
```

Comment:
Can you update this URL. It 404s at the moment.

> https://developer.bentley.com/api-groups/project-delivery/apis/validation/

Reply:
I see a few other 404ing URLS

Reply:
From Wil: 

"Yes the validation API has been significantly reworked since this sample was developed.  Until all these changes are available in PROD, it would be best if you just comment out the API call and have the sample rely on the JSON data file."

With this in mind, I've basically removed all the api calls.

Reply:
Ok. We should remove the links to documentation in the meantime.